pubspec.yaml

```diff
@@ -42,7 +42,7 @@ dependencies:
   toast: ^0.1.4                       # MIT
   image_picker: ^0.6.0+15             # Apache 2.0
   flutter_launcher_icons: ^0.7.2+1    # MIT
-  url_launcher: ^5.1.0                # BSD
+  url_launcher: ^5.1.2                # BSD
   keyboard_visibility: ^0.5.6         # MIT
```

Comment:
FYI I think changing this shouldn't make a different as the '^' means that it will update up to but not including 5.2.0.
Command might be something other than `flutter pub get` though